integrations/dalle/src/client.ts

```diff
@@ -0,0 +1,32 @@
+﻿import { AxiosResponse } from 'axios'
+import * as schemas from './schemas'
+import { ActionArgs } from './types'
+
+export function buildApiData(args: ActionArgs) {
```

Comment:
**suggestion:** Is there any user ID we can pass to easily detect abuse? not required but if it's an easy add I think we should do it, see `user` field : https://platform.openai.com/docs/api-reference/images/create